pipelines/services/run.go

```diff
@@ -36,7 +36,7 @@ func (rs *RunService) GetRunStatus(branch, pipeName string, isMultiBranch bool)
 	httpDetails := rs.getHttpDetails()
 
 	// Query params
-	queryParams := make(map[string]string, 0)
+	queryParams := make(map[string]string)
```

Comment:
Fixing :
G602 (CWE-118): Potentially accessing slice out of bounds (Confidence: MEDIUM, Severity: MEDIUM)